app/views/statuses/index.json.jbuilder

```diff
@@ -0,0 +1,7 @@
+json.statuses @statuses.each do |status|
+  json.id status.id
+  json.body status.body.length > 150 ? status.body[0...150] + '...' : status.body
```

Comment:
You could do two things here.

First would be to use the `truncate` method which receives an integer to slice the string and keep everything before it and also receive a string to concatenate at the end, respecting the slice value you have chosen.

Second would be to replace all magic numbers with a constant variable so you could keep consistent across views and tests. `TRUNCATE_BODY_LIMIT` would have 150 as its value.

```suggestion
  json.body status.body.truncate(Status::TRUNCATE_BODY_LIMIT, omission: '...')
```